### ISSUE TITLE
adding icon to the event 

### ISSUE THREAD

**Issue by Cbosh** [alamkanak/Android-Week-View#217]
I want to add the icon to the event for status purpose but the event can only display the subject

**Comment by caske33** [comment]
I think you'll have to adapt the library yourself to your need. But don't worry, the library is rather small, so it should probably be an easy fix.

**Comment by Cbosh** [comment]
if I adopt the library and make the changes, how do I add the updated one
back to my project.

**Comment by caske33** [comment]
you'll have to include this project als a gradle module by using a `project(:weekview:library)` instead of a `compile ...` (assuming the library is in the `weekview` submap, next to your `app` directory).

**Comment by lolobosse** [comment]
Yes, and I maybe it could be a good idea to use these fonts Icon to do that: you already have the possibility to write text so why not using a custom font which will manage the different resolution: I made it with [that](http://fontastic.me) for instance. But there are other ways, only a recommandation :smile:

**Comment by vermotr** [comment]
I created a PR that adds this functionality https://github.com/alamkanak/Android-Week-View/pull/266 ;-)

**Comment by Cbosh** [comment]
Thanks man, so should I adopt the library to use the methods that you created or I can be able to access the methods if I import it again

**Comment by vermotr** [comment]
I have proposed a PR to add this functionality but if you already want to use it, you can clone my fork and import it in Android Studio (File > New > Import Module...).

**Comment by isairuiz** [comment]
Hi everyone, how can i get this new feature in my project? I already have the library in my app but i want to add an icon my events. Thank you!
